### ISSUE TITLE
Support for RTMP protocol

### ISSUE THREAD

**Issue by Kantaris** [Kantaris/PlaySE-Smart-TV-app#9]
Some content on SVT Play is no longer available with the HLS protocol. This content  is produced by Showtime and it is possible that they have demanded better copy protection than what HLS gives. One popular show not available in HLS is Homeland.

The content is streamed using the RTMP protocol but it's still possible to play this content in the PlaySE app by embedding the official SVT Play Flash player within an iframe.

This is even recommended by Samsung. See here: http://www.samsungdforum.com/Guide/tut00093/index.html

SVT makes this quite easy by having the tag type=embed. For example: http://www.svtplay.se/video/1528099/diktaturens-fangar?type=embed

But here comes the tricky part. Inside the app there is no mouse pointer to control the flash player. We need to be able to control the flash player using javascript. 

Luckily SVT has made that possible to by making an external interface of some of the functions in the flash player.
![extern](https://f.cloud.github.com/assets/4547868/1328992/928e552a-350e-11e3-8c09-e0ea19a7d947.png)

SVT uses this themselves in their remote control feature:
See here: http://www.svtplay.se/kontroll  

The code they use to access the external functions of the flash player is in a file named script-built.js which has been minified and is therefor very hard to follow.

I wasn't able to call the functions and get to work but I'm almost certain that it is possible. Any help would be much appreciated.

**Comment by magicus** [comment]
Do you believe that solving this would enable the remote control to play content in Play SE ? If so, the benefit of adding this support would increase.  While only a few shows need RTMP, adding support for the mobile remote control would be _really_ nice.
